### ISSUE TITLE
How to specify account?

### ISSUE THREAD

**Issue by lww** [beardon/apple-reporter#2]
Hey, I'm getting this error:
```
Unhandled rejection AppleReporterError: You have access to more than one account. Please specify an account number in your properties file or on the command line. To see a list of accounts, run the command getAccounts.
```

After running `getAccount()` I get my account numbers, but how do I specify a single one? I tried this but it doesn't work:
```
const reporter = new AppleReporter({
    userid: 'test@test.com',
    password: 'mypassword',
    account: 12345
});
```

**Comment by tyscorp** [comment]
I don't have an ITC user with more than one account to test this.

Could you please `npm i apple-reporter@1.2.0-alpha.0` and then pass in the account like you're doing so above and report back if it works?

**Comment by lww** [comment]
yep, works. Thanks!

**Comment by tyscorp** [comment]
Published as [v1.2.0](https://github.com/beardon/apple-reporter/releases/tag/v1.2.0)
